backend/connection.go

```diff
@@ -190,6 +196,50 @@ func (c *Connection) watchEval(action Action) {
 	}
 }
 
+func (c *Connection) fetchMember(action Action) {
+	memberID := action.Payload.(string)
+	member, err := c.hub.nomad.MemberWithID(memberID)
+	if err != nil {
+		log.Errorf("websocket: unable to fetch member %q: %s", memberID, err)
+		return
+	}
+
+	c.send <- &Action{Type: fetchedMember, Payload: member}
+}
+
+
+func (c *Connection) watchMember(action Action) {
+	memberID := action.Payload.(string)
+
+	defer func() {
+		c.watches.Remove(memberID)
```

Comment:
https://github.com/hashicorp/nomad/blob/master/client/rpcproxy/rpcproxy.go#L77-L78

That list is being returned here:

https://github.com/hashicorp/nomad/blob/master/command/agent/agent_endpoint.go#L142